tests/components/rfxtrx/test_init.py

```diff
@@ -1,5 +1,7 @@
 """The tests for the Rfxtrx component."""
 
+from tests.async_mock import call
```

Comment:
Run isort too.

Reply:
Hehe. Did in my phone. Probably shouldn't have.